x-pack/plugins/security_solution/public/management/pages/trusted_apps/store/utils.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const parseQueryFilterToKQL = (filter: string): string => {
+  if (!filter) return '';
+  const kuery = [`name`, `description`, `entries.value`, `entries.entries.value`]
```

Comment:
I wonder why needed to also define the `entries.entries.value`. We are not using nested conditions.

Reply:
It seems that trusted apps with a `Signature` type entry have the `Signature`.value in this nested object.
From I saw, Hash.value and Path.value fills in `entries.value` but Signature.value is in `entries.entries.value`.
I saw this on the saved object mapping: x-pack/plugins/lists/server/saved_objects/exception_list.ts
Let me know if I wrong but this was the way I found to be able to filter by `Signature.value`

Reply:
Ahhh. yes, I remember now. You are correct. From the UI, the user only defines the value for the signer, but when we store it, we also add on to the record that the certificate must be trusted.
👍